sdk/core/core-amqp/package.json

```diff
@@ -113,7 +113,7 @@
     "unit-test:node": "cross-env TS_NODE_FILES=true TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\":\\\"commonjs\\\"}\" nyc --reporter=lcov --reporter=text-lcov mocha -r ts-node/register -t 50000 --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- ./test/**/*.spec.ts",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
-  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/%40azure/amqp-common_1.0.0-preview.5/sdk/core/amqp-common",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/%40azure/core-amqp_1.0.0-preview.1/sdk/core/core-amqp",
```

Comment:
The updated link here won't work. The previous link used a "tag", we should use the "master" branch instead